classes/author/Repository.inc.php

```diff
@@ -239,4 +250,20 @@ public function getSubmissionAuthors(Submission $submission, bool $onlyIncludeIn
                 ->orderBy(Repo::author()->getCollector()::ORDERBY_ID)
         );
     }
+
+    public function reorderAuthors(int $publicationId, $authors) {
```

Comment:
Add a description of this method. In particular, mention that this is for setting a specific author order (rather than DAO::resetContributorsOrder). Maybe it's worth calling the method `setAuthorsOrder`.

Reply:
added the following description 

`Reorders the authors of a publication according to the given order of the authors in the provided author array`

Changed its name to `setAuthorsOrder`